src/RcppExports.cpp

```diff
@@ -66,22 +88,18 @@ RcppExport SEXP _finalsize_final_size_grps_cpp(SEXP contact_matrixSEXP,
       p_susceptibilitySEXP);
   Rcpp::traits::input_parameter<const Eigen::MatrixXd&>::type susceptibility(
       susceptibilitySEXP);
-  Rcpp::traits::input_parameter<const Rcpp::String&>::type solver(solverSEXP);
-  Rcpp::traits::input_parameter<const Rcpp::List&>::type control(controlSEXP);
-  rcpp_result_gen = Rcpp::wrap(
-      final_size_grps_cpp(contact_matrix, demography_vector, p_susceptibility,
-                          susceptibility, solver, control));
+  rcpp_result_gen = Rcpp::wrap(prepare_data(contact_matrix, demography_vector,
+                                            p_susceptibility, susceptibility));
   return rcpp_result_gen;
   END_RCPP
 }
```

Comment:
As discussed on slack, could you revert to the R version here please? And possibly integrate it in the main `final_size()` function, as suggested by Tim?

Reply:
Removed C++ fun and added R version to `final_size` function body see https://github.com/epiverse-trace/finalsize/pull/73/commits/b48f8d859b516acb5b66823b3ae7b9145602f368.

That said - could we please not expand the scope of PRs indefinitely - this PR is basically three stuck together, which will impact readability later.